lib/sdf-server/src/service/v2/admin/migrate_connections.rs

```diff
@@ -260,7 +260,6 @@ async fn migrate_connection(
             // inferred connections.
             if migration.explicit_connection_id.is_none() {
                 info!("Making inferred socket connection explicit due to issues");
```

Comment:
We should just delete this block right?

Reply:
No we should keep it for now as information about any sockets that needed migrated